lib/sqlalchemy/engine/result.py

```diff
@@ -329,6 +333,9 @@ def __setstate__(self, state: Dict[str, Any]) -> None:
             _tuplefilter=_tuplefilter,
         )
 
+    def _contains(self, value: Any, row: Row[Unpack[_RawRowType]]) -> bool:
```

Comment:
**Michael Bayer** (zzzeek) wrote:

I think we removed this separately so this method can be removed

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4986